src/MSBuild/Resources/Strings.resx

```diff
@@ -877,6 +877,105 @@
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_43_GetPropertySwitch" Visibility="Public">
+    <value>  -getProperty:propertyName,...
+                      Write out the value of the property or properties after
+                      evaluation, without executing the build, or if either
+                      the -targets option or the -getTargetResult option is
+                      used, write out the values after the build.
+    </value>
+    <comment>
+      LOCALIZATION: "-getProperty" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_44_GetItemSwitch" Visibility="Public">
+    <value>  -getItem:itemName,...
+                      Write out the value of the item or items after evaluation
+                      without executing the build, or if either the -targets
```

Comment:
nit: including metadata?

Reply:
I meant that that switch writes out the value of the item(s) _including any associated metadata_ after evaluation. Might be good to include that.